java/requirements.yaml

```diff
@@ -3,3 +3,23 @@ dependencies:
      version: ~6.3.5
      repository: '@stable'
      condition: postgresql.enabled
+   - name: job
+     version: ~0.3.0
+     repository: '@hmctspublic'
+     alias: smoke-tests
+     condition: smoke-tests.enabled
```

Comment:
also should we make it `tests.smoke.enabled` , so that others will be tests.smoke.cron.enabled and tests.functional.enabled, tests.functional.cron.enabled ? Again for avoiding '-'s and also yaml might look readable.
tests:
  smoke:
    enabled: true
    cron:
      enabled: true
  functional: 
    enabled: true
    cron:
      enabled: true